ts/client/src/client.ts

```diff
@@ -812,21 +797,90 @@ export class MangoClient {
   }
 
   public async getMangoAccount(
-    mangoAccount: MangoAccount | PublicKey,
```

Comment:
simplified this

Reply:
This might break existing users, but I agree being able to call this on a MangoAccount was weird